.github/workflows/test.yml

```diff
@@ -28,21 +27,9 @@ jobs:
       - name: checkout
         uses: actions/checkout@v3
         with: { submodules: recursive }
-      - name: setup conda environment
-        uses: conda-incubator/setup-miniconda@v2
+      - uses: prefix-dev/setup-pixi@v0.8.1
         with:
-          python-version: "${{ matrix.python }}"
-          miniforge-variant: Mambaforge
-      - name: install dependencies
-        shell: bash -l {0}
-        run: |
-          mamba env update --quiet -n test -f environment.yml
-          conda list
-      - name: install unitpackage
-        shell: bash -l {0}
-        run: |
-          pip install -e .
+          pixi-version: v0.28.2
       - name: doctest
-        shell: bash -l {0}
         run: |
-          pytest -n auto --doctest-plus ${{ matrix.remote-data == 'remote' && '--remote-data' || '' }} --doctest-modules unitpackage
+          pixi run -e ${{ matrix.environment }} ${{ matrix.remote-data == 'remote' && 'doctest-remote' || 'doctest' }}
```

Comment:
```suggestion
          pixi run -e ${{ matrix.environment }} doctest ${{ matrix.remote-data == 'remote' && '--remote-data' || '' }}
```

Reply:
You should just be able to append parameters but maybe I am missing something here?

Reply:
It's funny that I had done it that way before https://github.com/echemdb/unitpackage/pull/63/commits/1e3e6260425409e726a35b53c6f883c460889433
and the failed tests might be related to that line in the test. :D

Works like a charm also locally.